solutions/5.js

```diff
@@ -0,0 +1,8 @@
+const recursionFunct = (funct, num) =>{
+  if(num === 0){
```

Comment:
if(num > 0){
funct();
recursionFunct(funct, num-1);
}

****this way, you won't need to return anything. & code looks cleaner